README.md

```diff
@@ -9,56 +9,57 @@ An action that enables operating GitHub release via pull request. You send a pul
 #
 # commitCategories:
-#   - title: string               # Category title.
-#     parentOfMergeCommit: bool   # True is whether the commit is the parent commit of the matching merge commit. Default is false.
-#     contains: []string          # Matches if commit's subject is prefixed by one of the given values. Default is emtpy.
-#     prefixes: []string          # Matches if commit's body is containing one of the given values. Default is emtpy.
+#   - title: string                     # Category title.
+#     parentOfMergeCommit: bool         # True is whether the commit is the parent commit of the matching merge commit. Default is false.
+#     contains: []string                # Matches if commit's subject is prefixed by one of the given values. Default is emtpy.
+#     prefixes: []string                # Matches if commit's body is containing one of the given values. Default is emtpy.
 #
 #
 # # Config used while generating release note.
 #
 # releaseNoteGenerator:
-#   showAbbrevHash: bool          # Whether to include abbreviated hash value in release note. Default is false.
-#   showCommitter: bool           # Whether to include committer in release note. Default is true.
-#   useReleaseNoteBlock: bool     # Whether to use release note block instead of commit message. Default is false.
-#   commitExclude:                # Additional excludes applied while generating release note.
-#     parentOfMergeCommit: bool   # True is whether the commit is the parent commit of the matching merge commit. Default is false.
-#     prefixes: []string          # Matches if commit's subject is prefixed by one of the given values. Default is emtpy.
-#     contains: []string          # Matches if commit's body is containing one of the given values. Default is emtpy.
-#   commitInclude:                # Additional includes applied while generating release note.
-#     parentOfMergeCommit: bool   # True is whether the commit is the parent commit of the matching merge commit. Default is false.
-#     prefixes: []string          # Matches if commit's subject is prefixed by one of the given values. Default is emtpy.
-#     contains: []string          # Matches if commit's body is containing one of the given values. Default is emtpy.
+#   showAbbrevHash: bool                # Whether to include abbreviated hash value in release note. Default is false.
+#   showCommitter: bool                 # Whether to include committer in release note. Default is true.
+#   useReleaseNoteBlock: bool           # Whether to use release note block instead of commit message. Default is false.
+#   useDefaultMergeCommitParser: bool   # Whether to use pull request metadata instead of commit message. If useReleaseNoteBlock is also true, release note block of pull request is used. Otherwise pull request title is used. If this option is set, showAbbrevHash and showCommitter is ignored. Default is false.
```

Comment:
nit: `useDefaultMergeCommitParser` -> `usePullRequestData` ?

Reply:
I think it's important to indicate that the user can't use this option other than merge-commit.
That's why, How about that I'll fix this option name to  `usePullRequestData` and add comments like this?
```
Whether to use pull request metadata instead of commit message when using merge-commit. If useReleaseNoteBlock is also true, release note block of pull request is used. Otherwise pull request title is used. If this option is set, showAbbrevHash and showCommitter is ignored. Default is false.
```

Reply:
👍 . That comment looks nice to me. 💯 

So let's change to `usePullRequestMetadata` to be the same as its comment.